explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -717,16 +731,31 @@ object TargetTabContents extends TwoPanels:
                         .getOrElse(IO.unit)
                     else IO.unit
 
+                  case LocalClipboard.CopiedTargets(tids) =>
+                    (props.focused.obsSet, optViewAgwo).tupled
+                      .foldMap((obsIds, agwov) =>
+                        val undoContext    = UndoContext(props.listUndoStacks, agwov)
+                        // don't want to do anything if the target astersism already contains the targets
```

Comment:
Do you mean we don't put the same target twice in the obs?

Reply:
I know, crazy, right?

Reply:
Your comment made me realize this bit needed improvement. 😄